images/build_images

```diff
@@ -28,14 +28,23 @@ CSD_URL_TEMPLATE = 'http://{s3_bucket}/datacollector/{build}/csd/STREAMSETS-{0}.
 DEFAULT_S3_BUCKET = 'nightly.streamsets.com'
 DEFAULT_BUILD = 'latest'
 EARLIEST_SDC_VERSION_WITH_LEGACY_MANIFEST = (3, 0, 0, 0)
+EL_VERSIONS = ['el6', 'el7']
 IMAGE_NAME_TEMPLATE = 'streamsets/clusterdock:topology_cdh-streamsets_datacollector-{}'
 LEGACY_STAGE_LIBS_INDEX_URL_TEMPLATE = 'http://{s3_bucket}/datacollector/{build}/legacy/index.html'
-PARCEL_URL_TEMPLATE = 'http://{s3_bucket}/datacollector/{build}/parcel/STREAMSETS_DATACOLLECTOR-{0}-el6.parcel'
+PARCEL_URL_TEMPLATE = ('http://{s3_bucket}/datacollector/{build}/parcel/'
+                       'STREAMSETS_DATACOLLECTOR-{sdc_version}-{el_version}.parcel')
 PARCEL_MANIFEST_URL_TEMPLATE = 'http://{s3_bucket}/datacollector/{build}/parcel/manifest.json'
 # Name of service in CDH cluster.
 SDC_PRODUCT_NAME = 'STREAMSETS_DATACOLLECTOR'
 
 
+class DockerImage:
```

Comment:
If you need a container like this, can probably simplify things with a namedtuple instead of a class

Reply:
I'd also say the container name is a bit weird since "parcel URL" isn't usually something I'd think of as instrumental to a Docker image. Can we perhaps go with `ParcelDockerImage` just to not cause confusion with the other places we use `DockerImage` (e.g. in STF's `build.py`)?

Reply:
Done.